os/arch/arm/src/armv7-r/arm_assert.c

```diff
@@ -972,5 +980,19 @@ void up_assert(const uint8_t *filename, int lineno)
 	(void)boardctl(BOARDIOC_RESET, 0);
 #endif
 
+#ifdef CONFIG_FAULT_MGR
+	fault_queue = mq_open("fault_queue", O_WRONLY | O_CREAT,  0666, 0);
```

Comment:
Where is a decision of fault location? When we send fault manager a message, is it ok to call `_up_assert` later?

Reply:
I have moved the fault manager inside _up_assert only with macro protection.